adapter/internal/discovery/xds/semantic_versioning.go

```diff
@@ -36,34 +36,14 @@ func GetVersionMatchRegex(version string) string {
 func GetMajorMinorVersionRangeRegex(semVersion semantic_version.SemVersion) string {
 	majorVersion := strconv.Itoa(semVersion.Major)
 	minorVersion := strconv.Itoa(semVersion.Minor)
-	if semVersion.Patch == nil {
```

Comment:
shall we remove Patch property from the struct as well.

Reply:
Done with: https://github.com/wso2/product-microgateway/pull/3506/commits/f7728c1c1ba607eb752bc325a389bb645f7b37b1